storage_service/locations/views.py

```diff
@@ -46,6 +46,11 @@ def package_list(request):
     packages = Package.objects.all()
     return render(request, 'locations/package_list.html', locals())
 
+def package_fixity(request, uuid):
+    package = Package.objects.get(uuid=uuid)
+    log_entries = FixityLog.objects.filter(package=package)
+    return render(request, 'locations/fixity_results.html', locals())
```

Comment:
I know it's super convenient, but we're trying to move away from using `locals()` and towards an explicit dict of the variables used.